org.civicrm.reqangular/src/common/templates/tab-outer.html

```diff
@@ -0,0 +1,14 @@
+<div>
+  <!--Overrides the 'uib/template/tabs/tabset.html'
+to add '{{tabset.customHeaderClass}}' which is used for custom header class-->
+  <ul class="nav nav-{{tabset.type || 'tabs'}} {{tabset.customHeaderClass}}"
+      ng-class="{'nav-stacked': vertical, 'nav-justified': justified}"
```

Comment:
Here 2 space indentation only is used. What you are pointing out is the line which is maintained by IDE's for attributes(same with other HTML's)
![tab-outer_html_-_hr17_-____buildkit_build_hr17_](https://cloud.githubusercontent.com/assets/5058867/25266086/7c26bf06-268d-11e7-9ca2-624b911c65a9.png)